docs/adr/0001-daily-active-use-server-side-metrics-glean.md

```diff
@@ -0,0 +1,121 @@
+
+The goal is to measure DAU (and subsequently WAU & MAU) by emitting metrics from syncstorage-rs itself. This requires the following data:
+* User identifier (hashed_fxa_uid)
+* Timestamp
+* Platform (Desktop, Fenix, iOS, Other)
+* Device Family (Desktop, Mobioe, Tablet, Other)
+* Device ID (hashed_device_id) for opt-out/deletion
+ 
+## Decision Drivers
+
+1. Simplicity of implementation, not only for internal metric emission, but for processing and querying.
+2. Reduction of complexity and load on team to process and make sense of data. Ideally there is existing infrastructure to reduce the possible significant work to calculate DAU.
+3. Glean is an internal tool successfully used by many teams at Mozilla.
+4. The Glean team has extensive experience and can provide considerable support in establishing application metrics.
+5. Extensibility: ability to expand scope of metrics to other measurements and apply methodology to other services in Sync ecosystem.
+
+## Considered Options
+
+* A. Glean Parser for Rust - Contribute to glean team repo by implementing Rust server
+* B. Custom Glean Implementation - Our own custom implementation, internal only to Sync
+* C. StatsD and Grafana
+
+## Decision Outcome
+
+Chosen option:
+
+* A. "Glean Parser for Rust: for server-side measurement of DAU"
+
+In researching possible implementation methods, it became clear that many options did not offer us the ease and flexibility to reconcile the data after emission. This is why Glean is recommended as a clear frontrunner, due to its rich tooling in aggregating, querying, and visualizing data.
+
+However, this left an addition decision to either implement our own custom Glean code to emit "Glean-compliant" output, or to contribute to the Glean team's `glean_parser` repository for server-side metrics. The `glean_parser` is used for all server implementations of Glean, since the SDK is only available for client-side applications. Currently, Rust is not supported.
+
+The Glean team does/did not have capacity to implemented the Rust `glean_parser` feature, so we had to decide what is the best solution not only for this use case, but to consider possible future use cases. In consultation with the Glean team, it became clear avoiding our custom implementation and opting for the general-purpose `glean_parser` for Rust was the ideal solution.
```

Comment:
nit: s/implemented/implement/

semantic nit: I'm not sure that the lack of capacity in the Glean team informs anything about future use cases.

The last sentence seems off to me. Something learned in consultation with the Glean team steered us to avoid a custom implementation, and that's why the glean parser route is chosen -- it's hinted at in "became clear" and such. I think we should just state plainly why the glean parser route was deemed better.